tests/baselines/reference/user/acorn.log

```diff
@@ -114,7 +114,6 @@ node_modules/acorn/dist/acorn.js(3032,40): error TS2339: Property 'inGeneratorCo
 node_modules/acorn/dist/acorn.js(3035,8): error TS2339: Property 'exprAllowed' does not exist on type 'TokenType'.
 node_modules/acorn/dist/acorn.js(5297,5): error TS2339: Property 'nextToken' does not exist on type 'Parser'.
 node_modules/acorn/dist/acorn.js(5298,12): error TS2339: Property 'parseExpression' does not exist on type 'Parser'.
-node_modules/acorn/dist/acorn_loose.es.js(1,56): error TS2440: Import declaration conflicts with local declaration of 'defaultOptions'.
```

Comment:
from @orta's change to allow property assignment declarations on imported things.